git_cli_vs_gui/git_cli_vs_gui.md

```diff
@@ -0,0 +1,337 @@
+module_type: standard
+docs_version: 4.0.0
+language: en
+narrator: US English Female
+mode: Textbook
+title: Git Command Line Interface versus Graphical User Interface
+comment:  Compare the two ways of interacting with Git to decide which is best for you.
+long_description: There are two ways of using Git: (1) typing commands on a command line interface (CLI) or (2) clicking through options on a graphical user interface (GUI) software. This module will compare and contrast these two options to help you decide how to use Git in your work.
+estimated_time_in_minutes: 30
+
+@pre_reqs
+Learners should know what version control is and why it's important and should also understand what a command line interface is and how that's different from a graphical user interface.  
+
+To acquire basic knowledge of version control, we recommend our [Introduction to Version Control](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/git_intro/git_intro.md#1) module. 
+
+To acquire basic knowledge about command line interfaces, we recommend our [Demystifying the Command Line Interface](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/demystifying_command_line/demystifying_command_line.md#1) module.  
+
+No previous experience with git is required.
+@end
+
+@learning_objectives  
+After completion of this module, learners will be able to:
+
+- List an advantage and a disadvantage of using Git via a **graphical user interface**
+- List an advantage and a disadvantage of using Git via a **command line interface**
+- Name one major Git graphical user interface 
+
+@end
+
+good_first_module: false
+data_task: data_management
+collection: infrastructure_and_technology
+coding_required: false
+sequence_name: git_basics
+previous_sequential_module: git_intro
+
+@sets_you_up_for
+
+- git_setup_mac_and_linux
+- git_setup_windows
+
+
+@end
+
+@depends_on_knowledge_available_in
+
+- git_intro
+- demystifying_command_line
+
+@end
+
+@version_history 
+Previous versions: 
+
+No previous versions.
+
+@end
+
+import: https://raw.githubusercontent.com/arcus/education_modules/main/_module_templates/macros.md
+-->
+
+# Git Command Line Interface versus Graphical User Interface
+
+@overview
+
+## What is Git?
+
+You may already know what Git is and even use it for version control, but let's review.  We'll take on some of the text that Git uses to describe itself in its official website:
+
+[![The official git website, which states: "Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency. Git is easy to learn and has a tiny footprint with lightning fast performance. It outclasses SCM tools like Subversion, CVS, Perforce, and ClearCase with features like cheap local branching, convenient staging areas, and multiple workflows."](media/git_scm_com.png)](https://git-scm.com)
```

Comment:
@leemc-data-ed I ended up taking out "text" as well in this alt text... what say?

Reply:
I like it!